guava-shaded/pom.xml

```diff
@@ -78,6 +74,8 @@
                 <includes>
                   <include>org.apache.cassandra:java-driver-guava-shaded</include>
                   <include>com.google.guava:guava</include>
+                  <include>com.google.guava:failureaccess</include>
```

Comment:
New dependency introduced in [Guava 27.0](https://github.com/google/guava/releases/tag/v27.0).